src/utils/hotkeys.py

```diff
@@ -0,0 +1,354 @@
+import os
+from enum import Enum
+from config import Config
+from ui.skills import get_selected_skill
+from screen import convert_screen_to_monitor, grab
+import template_finder
+import keyboard
+from utils.custom_mouse import mouse
+from utils.misc import wait
+from logger import Logger
+
+d2r_keymap = {}
```

Comment:
Please add a short comment on what these are. Only needed for the "public" ones. Thus if anybody wants to use this data, they can look it up here.

Reply:
Comments have been added for describing the variables